contracts/voting/dao-voting-native-staked/src/contract.rs

```diff
@@ -66,10 +66,18 @@ pub fn instantiate(
     CONFIG.save(deps.storage, &config)?;
     DAO.save(deps.storage, &info.sender)?;
 
+    // Validate denom by checking supply
+    let supply: Coin = deps.querier.query_supply(msg.denom.to_string())?;
+    println!("supply {:?}", supply);
```

Comment:
The fix looks good to me! There is just a println remaining in line 71 that looks like its left over from debugging